app/models/production_release.rb

```diff
@@ -142,6 +142,7 @@ def fetch_health_data!
     release_data = monitoring_provider.find_release(platform, version_name, build_number, store_rollout.created_at)
     return if release_data.blank?
     release_health_metrics.create!(fetched_at: Time.current, **release_data)
+    HaltUnhealthyReleaseRolloutJob.perform_async(id)
```

Comment:
thought: `ReleaseHealthEvent` has this thing 

```ruby
after_create_commit :notify_health_rule_triggered
```

which notifies, I think similarly we can inline halting in `ReleaseHealthEvent` or move it to a `Signal` instead.

but more importantly, we should also mark this debug-flag we'd added called `action_triggered` which we should mark as `true` once we kickoff the job, so that there's a log of the fact that a particular health event caused an action to happen.

Reply:
additionally, we need to do the reverse as well. when a release health event makes the release healthy, we should "resume" the release from halted

currently we just wait for the next rollout to kick in (which is good) but halting "stops" the adoption of the app, so when it's healthy it should restart the adoption of the app as well (before it increments) if possible.